etc/shinken.cfg

```diff
@@ -79,7 +79,7 @@ disable_old_nagios_parameters_whining=0
 #use_timezone=Europe/Paris
 
 # Disabling env macros is good for performances. If you really need it, enable it.
-enable_environment_macros=0
+enable_environment_macros=1
```

Comment:
oups oups oups: this is a very bad thing to enable globaly, really. This made communication and CPU explode on schedulers.